packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -1,28 +1,23 @@
-const syncAssets = require('../sync-conflict-handler-assets/syncAssets');
-const inquirer = require('inquirer');
-const fs = require('fs-extra');
-const uuid = require('uuid');
-const path = require('path');
-const open = require('open');
-const { rootAssetDir } = require('../index');
-const TransformPackage = require('graphql-transformer-core');
-
-const category = 'api';
-const serviceName = 'AppSync';
-const parametersFileName = 'parameters.json';
-const schemaFileName = 'schema.graphql';
-const providerName = 'awscloudformation';
-const resolversDirName = 'resolvers';
-const stacksDirName = 'stacks';
-const defaultStackName = 'CustomResources.json';
-const graphqlSchemaDir = path.join(rootAssetDir, 'graphql-schemas');
-
-const {
+import { ListQuestion, CheckboxQuestion, ListChoiceOptions } from 'inquirer';
+import { dataStoreLearnMore } from '../sync-conflict-handler-assets/syncAssets';
+import inquirer from 'inquirer';
+import fs from 'fs-extra';
+import uuid from 'uuid';
+import path from 'path';
+import open from 'open';
+import { rootAssetDir, parametersFileName } from '../awsConstants';
+import {
   collectDirectivesByTypeNames,
   readTransformerConfiguration,
   writeTransformerConfiguration,
-  TRANSFORM_CURRENT_VERSION,
-} = TransformPackage;
+  readProjectConfiguration,
+} from 'graphql-transformer-core';
+import { category } from '../../../categoryConstants';
+import { writeResolverConfig } from '../CfnApiArtifactHandler';
+
+const serviceName = 'AppSync';
```

Comment:
Does this change the serviceName we store in amplify-meta? If so - this change might break existing projects. We use the service-name in a variety of packages including code-gen.

Reply:
No, this is the existing service name and I changed the service name enum in the spec to match this

Reply:
Does this change the service name which we store in the amplify-meta though?

Reply:
no, "AppSync" is what we currently store in amplify-meta